### PR TITLE
Mosip 25703 reg client display both sdk and mds score

### PR DESCRIPTION
**Pull request by Rediket**

### CODE REVIEW

File: registration/registration-services/src/main/java/io/mosip/registration/dto/RegistrationDTO.java [hunk 1]
```diff
@@ -245,7 +247,15 @@ public void removeExceptionPhoto(String fieldId) {
 		    	iterator2.remove();
 		    }
 		}
-		
+
+		Iterator<Entry<String, Double>> iterator3 = this.SDK_SCORES.entrySet().iterator();
```
Comment (ase-101) [inline registration/registration-services/src/main/java/io/mosip/registration/dto/RegistrationDTO.java:251]: let's rename 
"iterator" to bioCapturesIterator
"iterator2" to bioScoresIterator
"iterator3" to sdkScoresIterator

File: registration/registration-client/src/main/java/io/mosip/registration/controller/device/GenericBiometricsController.java [hunk 2]
```diff
@@ -300,7 +301,9 @@ private void displayBiometric(Modality modality) {
 		applicationLabelBundle = ApplicationContext.getBundle(applicationContext.getApplicationLanguage(), RegistrationConstants.LABELS);
 
 		retryBox.setVisible(!isExceptionPhoto(modality));
-		thresholdBox.setVisible(!isExceptionPhoto(modality));
+		thresholdBox.setVisible(!isExceptionPhoto(modality) && isQualityCheckWithSdkEnabled());
```
Comment (ase-101) [inline registration/registration-client/src/main/java/io/mosip/registration/controller/device/GenericBiometricsController.java:304]: Is this code tested after disabling "QUALITY_CHECK_WITH_SDK"?
"thresholdBox" is a gridpane and inside that we display mdsScore and SDKScore. if "QUALITY_CHECK_WITH_SDK" is set to false, then the whole "thresholdBox" gridpane is hidden.
Can you pls cross check?

**Comment by ase-101** [comment]
And also @Rediket make sure you add **mdsQuality** and **sdkQuality** labels here https://github.com/mosip/artifactory-ref-impl/tree/develop/artifacts/src/i18n/regclient-i18n-bundle

File: registration/registration-client/src/main/java/io/mosip/registration/controller/device/GenericBiometricsController.java (hunk 2 above)
Comment (Rediket) [reply registration/registration-client/src/main/java/io/mosip/registration/controller/device/GenericBiometricsController.java:301]: Yes, I have tested it. If "QUALITY_CHECK_WITH_SDK" is set to 'N' then it is hidden. The next line of code: `GridPane.setMargin(thresholdBox, new Insets(0, 0, isQualityCheckWithSdkEnabled() ? 50 : 0,0))` adjusts the height of the thresholdBoxdepending on whether or not the presence of the quality sdk.

**Review by ase-101 — APPROVED** [review]

**Pull request merged**